crates/turborepo-lib/src/daemon/server.rs

```diff
@@ -255,6 +258,9 @@ impl<T: Watcher + Send + 'static> proto::turbod_server::Turbod for DaemonServer<
     ) -> Result<tonic::Response<proto::NotifyOutputsWrittenResponse>, tonic::Status> {
         let inner = request.into_inner();
 
+        self.times_saved
+            .insert(Arc::new(inner.hash).to_string(), inner.time_saved);
```

Comment:
I think this is what you want
```suggestion
            .insert(inner.hash.clone(), inner.time_saved);
```